### PR TITLE
Layout

### PR DESCRIPTION
**Pull request by ismailakinkunmi**
- Keep the general layout of the wireframe and styling

### CODE REVIEW

**Review by elisha2kyakpo1 — CHANGES_REQUESTED** [review]
Hi @ismailakinkunmi,

Great job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._
- Let's make sure that all the necessary commands required for the app to run on a users local machine are listed in the readme file.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ismailakinkunmi** [comment]
Hi Elisha, please did you mean I should add npm start to Readme file also what did
you mean by x,y,z

**Comment by elisha2kyakpo1** [comment]
Hey @ismailakinkunmi, commands required to run the app on the terminal, npm start is one of them. As for the zyz, ignore those, that was my error.

File: README.md [hunk 1]
```diff
@@ -17,13 +17,15 @@ This site was built using [GitHub Pages](https://https://ismailakinkunmi.github.
...
 
-git clone git@github.com:ismailakinkunmi/leaderboard.git
+- git clone git@github.com:ismailakinkunmi/leaderboard.git
 
-cd to the folder
+- cd to the folder
 
-cd Project-Webpack
+- cd Project-Leaderboard
+
```
Comment (t-yanick) [inline README.md:27]: - [OPTIONAL] Please you have to install dependencies first by running `npm i` in the console, please kindly update that information in your `readme file`.

**Review by t-yanick — APPROVED** [review]
Hi @ismailakinkunmi, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (ismailakinkunmi) [reply README.md:27]: @t-yanick thanks for the feedback.

**Pull request merged**